components/extractors/other/telemetry-data/telemetry-data.md

```diff
@@ -380,6 +380,8 @@ organization, or to the Keboola Connection platform itself.
 |`auditLog.maintainers.listed`
 |`auditLog.maintainers.organizationCreated`
 |`auditLog.maintainers.updated`
+|`auditLog.mergeRequest.created`
```

Comment:
Prověřoval jsem to proti https://github.com/keboola/event-schema/pull/14/files a zkontroloval jsem, že approvalReceived eventa co se tam přidala tady nakonec není. Bylo by dobré ji tam odstranit taky.

Reply:
neni, protoze se nestrili (nakonec jsem zjistil, ze to neni potreba). Mam to v planu z event-schema zrusit